README.md

```diff
@@ -58,96 +58,95 @@ pip install jupyterhub-ltiauthenticator
     is available to help. Your LMS provider might also implement custom keys
     you can use.
 
-5.  Pick a name for edX to call your JupyterHub server. Then, along with the two random strings you generated in step 4, paste them together to create an _LTI Passport String_ in the following format:
+- **consumers**: the value for the LTI 1.1 configuration consumer key and shared secret created in Step 1 of [the common configuration settings section](#common-configuration-settings).
+- **username_key**: the value for the LTI 1.1 configuration XML description.
```

Comment:
Comments that seem to relate to the above configuration step in the main bullet point list, so should perhaps be indented also?